secure_redis/client.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
```

Comment:
I understand the logic here. I think (for now) we should not include a client in the OSS software, but we should definitely call this out as a warning in the README. incrementing is probably ok to just have pass through without encryption (at least for now) since we don't use this, and its rather tough to support anyway.

If we get users of this software, we can see if there is an interest in supporting this, and discuss how to do it with them in the future. Please remove.